CollAction/Services/ProjectService.cs

```diff
@@ -212,6 +221,40 @@ private string FormatEmailMessage(string message, ApplicationUser user, string u
             return message;
         }
 
+        private static string ToUrlSlug(string value)
+        {                      
+            value = value.ToLowerInvariant();
+            value = _spaceRemoveRegex.Replace(value, "-");
+            value = _invalidCharRemoveRegex.Replace(value, "");
+            value = value.Trim('-', '_');
+            value = _doubleDashRemoveRegex.Replace(value, "$1");
+            if(value.Length == 0)
+            {
+                value = "-";
+            }
+             return value;
+        }
+        private static string RemoveDiacriticsFromString(string text) 
+        {
+            var normalizedString = text.Normalize(NormalizationForm.FormD);
+            var stringBuilder = new StringBuilder();
+             foreach (var c in normalizedString)
+            {
+                var unicodeCategory = CharUnicodeInfo.GetUnicodeCategory(c);
+                if (unicodeCategory != UnicodeCategory.NonSpacingMark)
+                {
+                        stringBuilder.Append(c);
+                }
+            }
+             return stringBuilder.ToString().Normalize(NormalizationForm.FormC);
+        }
+        public string GetProjectNameNormalized(string projectName)
```

Comment:
Needs an empty line between methods. You still haven't fixed that on the other pull request (289) as well.